Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:1.9
+
+EXPOSE 9292
+
+WORKDIR /pg_web_stats
+COPY . /pg_web_stats/
+RUN set -ex; \
+    mkdir /etc/pg_web_stats; \
+    ln -sf /etc/pg_web_stats/config.yml /pg_web_stats/config.yml; \
```

Comment:
In that case you also wouldn't need to rebuild docker imagine to use it with another database

Reply:
but backend does not support that

Reply:
I'm planning to push one more PR with multiple databases support(like in https://github.com/ankane/pghero).
Environment variable will become useless then.

Reply:
The image does not need to be rebuilt. Users just need to map the configuration file into a container (as a bind-mount, or e.g. using Docker or Kubernetes secrets).

I suppose, just like with PgHero, a `DATABASE_URL` is useful - even with multi-DB support we plan to add, it can be a neat shortcut to say "there is just one database". However, I feel that this is a new feature that's really out of scope of this PR (and my #7) which is essentially about just packaging existing codebase into a Docker image.